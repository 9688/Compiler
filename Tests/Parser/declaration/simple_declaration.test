int x1;
char x2;
double x3;
int x, y, z;