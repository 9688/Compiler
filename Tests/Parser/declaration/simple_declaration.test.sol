<<<table #1>>>
<tags>
<types>
int
char
double
void
<vars>
var printf type function printf(var  type  to char, ){
} returned void
var scanf type function scanf(var  type  to char, ){
} returned void
var getchar type function getchar(){
} returned int
var x1 type int
var x2 type char
var x3 type double
var x type int
var y type int
var z type int

