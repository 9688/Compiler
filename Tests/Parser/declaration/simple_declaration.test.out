<<<table #1>>>
<tags>
<types>
int
char
double
void
<vars>
var x1 type int
var x2 type char
var x3 type double
var x type int
var y type int
var z type int

